### DISCUSSION TITLE
How to get concave hull？

### DISCUSSION THREAD

**Discussion by jcwwc** [CGAL/cgal#5976] [Q&A]
_Please use the following template to help us solving your issue._

## Issue Details

I want to know how to get the concave hull boundary of point cloud. 
I tried alpha Shape, but I still got convex hull.

## Source Code

This is the official instance code：
CGAL-5.1.3\examples\Alpha_shapes_2

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): windows + vs2017
* Compiler:
* Release or debug mode: release & debug
* Specific flags used (if any):
* CGAL version: 5.13
* Boost version: 1.64
* Other libraries versions if used (Eigen, TBB, etc.): pcl qt

**Comment by sloriot** [comment]
Did you see that page?
https://doc.cgal.org/latest/Manual/tuto_reconstruction.html
